README.md

```diff
@@ -1,3 +1,24 @@
 # CCDL Training Modules
 
-A collection of modules that are combined into 1-5 day workshops on computational topics for the childhood cancer research community.
+This repository is a collection of modules that are combined into 1-5 day workshops on computational topics for the childhood cancer research community.
+
+**If you are childhood cancer researcher** and are interested attending a workshop go to the [CCDL's website](https://www.ccdatalab.org/projects-training)
```

Comment:
Just a couple minor additions, "If you are **a** childhood cancer researcher and are interested **in** attending..."